libs/full/performance_counters/include/hpx/performance_counters/counters.hpp

```diff
@@ -242,8 +242,8 @@ namespace hpx { namespace performance_counters {
     /// instance. Generally, a full name of a counter instance has the
     /// structure:
     ///
-    ///    /objectname{parentinstancename#parentindex/instancename#instanceindex}
-    ///      /countername#parameters
+    ///    /objectname{parentinstancename#parentindex/instancename}
+    ///      /countername
```

Comment:
What was doxygen complaining about here?

Reply:
The warnings are:

- `explicit link request to 'instanceindex' could not be resolved`

- `explicit link request to 'parameters' could not be resolved`